src/subscription_manager/repofile.py

```diff
@@ -469,6 +472,10 @@ def fix_content(self, content):
             apt_cont['Suites'] = 'default'
             apt_cont['Components'] = 'all'
             apt_cont['Trusted'] = 'yes'
+
+            if 'arches' in apt_cont and apt_cont['arches'][0] != 'ALL':
+                apt_cont['Architectures'] = " ".join(apt_cont['arches'])
```

Comment:
The default value is set using `self.PROPERTIES['arches']` on line 76. When `apt_cont['arches']` contains this default value: '0', then this string is is used for: `apt_cont['Architectures']` too.

Reply:
Is `apt_cont['Architectures']` supposed to replace `apt_cont['arches']`?

Reply:
Hey @jirihnidek, I take this PR over from @sbernhard.
`'arches'` is meant to be more general and contains the list of architectures as they are sent from candlepin (compare [line 157](https://github.com/candlepin/subscription-manager/pull/2213/files#diff-bba0de40e76155c729439fe156734acb0668464c5c8ade3cc7f2560944aa586bR157)). In the more general part of this module (e.g. `Repo`) we don't use the Debian-specific parameter `'Architectures'`, in order to be coherent with the other default parameters (as they are all written in small letters). Moreover, this gives the ability to use the same parameter `'arches'` for other RepoFile classes, if they are about to add a parameter to describe the repo architecture too.
Therefore, `Architectures` is added in this section of `AptRepoFile` when `fix_content` is called.

For now, both parameters (`'arches'` and `'Architectures'`) are written to the `rhsm.sources` file. It doesn't make any difference, since apt ignores unknown parameters. Nevertheless, would you suggest removing `'arches'` from `apt_content` in this section to leave it out of `rhsm.sources`?